docs/dag_scheduler.md

```diff
@@ -31,6 +31,16 @@ There are two special kinds of nodes - Request and Response node. Both of them a
     Each model input needs to be mapped to some node's `data_item` - input from gRPC/REST `request` or another `DL model` output. 
     Outputs of the node may be mapped to another node's inputs or the `response` node, meaning it will be exposed in gRPC/REST response. 
 
+### Custom node type
+
+* custom - that node can be used to implement all operations on the data which can not be handled by the neural network model. It is represented by
+a C++ dynamic library implementing OVMS API defined in [custom_node_interface.h](../src/custom_node_interface.h). Custom nodes can run the data
+processing using OpenCV, which is included in OVMS, or in could include other third-party components. Custom node libraries are integrated with OVMS
+ by adding its definition to the pipeline configuration. The configuration includes a path to the compiled binary with `.so` extension. 
+Unlike models, custom nodes are not versioned.
```

Comment:
I don't think this brings anything. 

We mention that pipelines are not versioned, custom nodes are extension of pipelines.

Reply:
I think it is a relevant information that custom node libs can now be versioned like models. This is also a difference with triton

Reply:
In doc custom node libs are not versioned. Either I would skip this information or provide  instruction if someone wants to use new version of library.

Reply:
"Custom nodes are not versioned, meaning one custom node library is bound to one name. To load another version, another name needs to be used."